tests/storage/test_id_generators.py

```diff
@@ -233,23 +237,23 @@ async def _get_next_async():
         # ... but calling `get_next` on the second instance should give a unique
         # stream ID
 
-        async def _get_next_async():
+        async def _get_next_async2() -> None:
```

Comment:
This is to avoid an error about re-declaring or something?

Reply:
Yes, it is to avoid re-declaring. There is a different function in the same test case with the same name: `_get_next_async()`.